cmd/wasm-client/main.go

```diff
@@ -31,53 +33,32 @@ import (
 type stubPackageRegistration func(map[string]func(context.Context,
 	*grpc.ClientConn, string, func(string, error)))
 
-var (
-	cfg     = config{}
-	lndConn *grpc.ClientConn
-
-	registry = make(map[string]func(context.Context, *grpc.ClientConn,
-		string, func(string, error)))
-
-	registrations = []stubPackageRegistration{
-		lnrpc.RegisterLightningJSONCallbacks,
-		lnrpc.RegisterStateJSONCallbacks,
-		autopilotrpc.RegisterAutopilotJSONCallbacks,
-		chainrpc.RegisterChainNotifierJSONCallbacks,
-		invoicesrpc.RegisterInvoicesJSONCallbacks,
-		routerrpc.RegisterRouterJSONCallbacks,
-		signrpc.RegisterSignerJSONCallbacks,
-		verrpc.RegisterVersionerJSONCallbacks,
-		walletrpc.RegisterWalletKitJSONCallbacks,
-		watchtowerrpc.RegisterWatchtowerJSONCallbacks,
-		wtclientrpc.RegisterWatchtowerClientJSONCallbacks,
-		looprpc.RegisterSwapClientJSONCallbacks,
-		poolrpc.RegisterTraderJSONCallbacks,
-	}
-)
+var registrations = []stubPackageRegistration{
+	lnrpc.RegisterLightningJSONCallbacks,
+	lnrpc.RegisterStateJSONCallbacks,
+	autopilotrpc.RegisterAutopilotJSONCallbacks,
+	chainrpc.RegisterChainNotifierJSONCallbacks,
+	invoicesrpc.RegisterInvoicesJSONCallbacks,
+	routerrpc.RegisterRouterJSONCallbacks,
+	signrpc.RegisterSignerJSONCallbacks,
+	verrpc.RegisterVersionerJSONCallbacks,
+	walletrpc.RegisterWalletKitJSONCallbacks,
+	watchtowerrpc.RegisterWatchtowerJSONCallbacks,
+	wtclientrpc.RegisterWatchtowerClientJSONCallbacks,
+	looprpc.RegisterSwapClientJSONCallbacks,
+	poolrpc.RegisterTraderJSONCallbacks,
+}
 
 func main() {
 	defer func() {
 		if r := recover(); r != nil {
-			log.Debugf("Recovered in f: %v", r)
+			fmt.Printf("Recovered in f: %v", r)
```

Comment:
nit: perhaps add a comment here that logging might not be initialized yet and we don't want to panic on a panic?